### ISSUE TITLE
LDAP login failure

### ISSUE THREAD

**Issue by aboettger-tuhh** [hedgedoc/hedgedoc#2561]
### Description

LDAP login not longer works in 1.9.4

### Steps to reproduce

1. Upgrade from 1.9.3  to 1.9.4
2. Login via LDAP
3. `Type Error` in Log

### Expected behaviour

Working Login as is in 1.9.3

### Logs

```shell
Aug 08 11:51:04 flint docker-compose[8305]: app_1       | TypeError: Cannot add property 2, object is not extensible
Aug 08 11:51:04 flint docker-compose[8305]: app_1       |     at Array.push (<anonymous>)
Aug 08 11:51:04 flint docker-compose[8305]: app_1       |     at LdapAuth._findUser (/hedgedoc/node_modules/ldapauth-fork/lib/ldapauth.js:329:21)
Aug 08 11:51:04 flint docker-compose[8305]: app_1       |     at LdapAuth.authenticate (/hedgedoc/node_modules/ldapauth-fork/lib/ldapauth.js:418:8)
Aug 08 11:51:04 flint docker-compose[8305]: app_1       |     at Strategy.handleAuthentication (/hedgedoc/node_modules/passport-ldapauth/lib/passport-ldapauth/strategy.js:276:8)
Aug 08 11:51:04 flint docker-compose[8305]: app_1       |     at Strategy.authenticate (/hedgedoc/node_modules/passport-ldapauth/lib/passport-ldapauth/strategy.js:344:33)
Aug 08 11:51:04 flint docker-compose[8305]: app_1       |     at attempt (/hedgedoc/node_modules/passport/lib/middleware/authenticate.js:369:16)
Aug 08 11:51:04 flint docker-compose[8305]: app_1       |     at authenticate (/hedgedoc/node_modules/passport/lib/middleware/authenticate.js:370:7)
Aug 08 11:51:04 flint docker-compose[8305]: app_1       |     at /hedgedoc/lib/web/auth/ldap/index.js:88:5
Aug 08 11:51:04 flint docker-compose[8305]: app_1       |     at Layer.handle [as handle_request] (/hedgedoc/node_modules/express/lib/router/layer.js:95:5)
Aug 08 11:51:04 flint docker-compose[8305]: app_1       |     at next (/hedgedoc/node_modules/express/lib/router/route.js:144:13)
Aug 08 11:51:04 flint docker-compose[8305]: app_1       |     at /hedgedoc/node_modules/body-parser/lib/read.js:137:5
Aug 08 11:51:04 flint docker-compose[8305]: app_1       |     at AsyncResource.runInAsyncScope (node:async_hooks:202:9)
Aug 08 11:51:04 flint docker-compose[8305]: app_1       |     at invokeCallback (/hedgedoc/node_modules/raw-body/index.js:231:16)
Aug 08 11:51:04 flint docker-compose[8305]: app_1       |     at done (/hedgedoc/node_modules/raw-body/index.js:220:7)
Aug 08 11:51:04 flint docker-compose[8305]: app_1       |     at IncomingMessage.onEnd (/hedgedoc/node_modules/raw-body/index.js:280:7)
Aug 08 11:51:04 flint docker-compose[8305]: app_1       |     at IncomingMessage.emit (node:events:539:35)
Aug 08 11:51:04 flint docker-compose[8305]: app_1       |     at endReadableNT (node:internal/streams/readable:1345:12)
Aug 08 11:51:04 flint docker-compose[8305]: app_1       |     at processTicksAndRejections (node:internal/process/task_queues:83:21)

```

### Config

```shell
CMD_LDAP_PROVIDERNAME: 'TU Hamburg'
CMD_LDAP_SEARCHATTRIBUTES: 'uid, mail'
CMD_LDAP_SEARCHBASE: 'ou=People,dc=tu-harburg,dc=de'
CMD_LDAP_SEARCHFILTER: '(uid={{username}})'
CMD_LDAP_URL: 'ldaps://ldap.rz.tu-harburg.de'
```

### Your Setup

- quay.io/hedgedoc/hedgedoc:1.9.4

**Comment by davidmehren** [comment]
Thanks for your report!
This turned out to be a bad interaction between us freezing the config object and the LDAP library trying to change it since a patch-release that made it into HedgeDoc 1.9.4.

As soon as #2583 is merged, you can try out the fix in the next [nightly Docker image](https://quay.io/repository/hedgedoc/hedgedoc-nightly)

**Comment by ErikMichelson** [comment]
As the fix #2583 got merged and released with 1.9.5, I'm closing this issue.
